src/__tests__/fixtures/component_36.js

```diff
@@ -0,0 +1,16 @@
+import { Sizes } from '../common';
+import T from 'prop-types';
+
+function SuperDuperCustomButton() {
+  return <div />;
+}
+
+SuperDuperCustomButton.defaultProps = {
+  size: Sizes.EXTRA_LARGE,
+};
+
+SuperDuperCustomButton.propTypes = {
+  size: T.oneOf(Object.values(Sizes)),
```

Comment:
Haven't looked at the implementation yet, but this snapshot reflects the current behaviour. Is it safe to assume this PR does not (yet) handle this behaviour, but it will given this test was checked in? I'm expecting both of these references to `Sizes` to resolve to a value rather than those string literals.

Also, `../common` is not implemented - not sure if on purpose or not.